### PR TITLE
cleanup: Remove VoxelPrimitive.jitter option (#11913)

### PR DESCRIPTION
**Pull request by okrayum**
# Description

In VoxelPrimitive.js I removed the following code blocks:

/**
   * @type {boolean}
   * @private
   */
  this._jitter = false;
  
  
  /**
   * Gets or sets whether or not to jitter the view ray during the raymarch.
   * This reduces stair-step artifacts but introduces noise.
   *
   * @memberof VoxelPrimitive.prototype
   * @type {boolean}
   */
  jitter: {
    get: function () {
      return this._jitter;
    },
    set: function (jitter) {
      //>>includeStart('debug', pragmas.debug);
      Check.typeOf.bool("jitter", jitter);
      //>>includeEnd('debug');

      if (this._jitter !== jitter) {
        this._jitter = jitter;
        this._shaderDirty = true;
      }
    },
  },

In VoxelRenderResources.js I removed the following code block:

if (primitive._jitter) {
    shaderBuilder.addDefine("JITTER", undefined, ShaderDestination.FRAGMENT);
  }

In VoxelPrimitiveSpec.js I removed the following line from the test "toggles render options that require shader rebuilds":

toggleOption("jitter", false, true);
  

This was a cleanup issue #11913

## Issue number and link

Issue #11913

## Testing plan

I ran npm run test prior to making any changes in the codebase and it showed 11 failing tests, I ran it again after making changes and it had the same failing tests. This was all done in the terminal, I then ran the tests through the browser, and it showed one failing test. I'm not sure if that is just in my environment. If they fail in the workflow I may need some additional guidance.

# Author checklist

- [ x] I have submitted a Contributor License Agreement
- [ x] I have added my name to `CONTRIBUTORS.md`
- [ x] I have updated `CHANGES.md` with a short summary of my change
- [ x] I have added or updated unit tests to ensure consistent code coverage
- [ ] I have updated the inline documentation, and included code examples where relevant
- [ x] I have performed a self-review of my code

If approved this will close issue #11913

### CODE REVIEW

**Comment by okrayum** [comment]
This is ready for workflow checks.

**Comment by ggetz** [comment]
Thanks @okrayum! @ankittrehan2000 will be reviewing this PR.

File: CHANGES.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 - Exposes `ScreenSpaceCameraController.zoomFactor` to allow adjusting the zoom factor (speed). [#9145](https://github.com/CesiumGS/cesium/pull/9145)
 
+- Removed `jitter` option from `VoxelPrimitive.js` and related test code in `VoxelPrimitiveSpec.js`. [#11913](https://github.com/CesiumGS/cesium/issues/11913)
```
Comment (ankittrehan2000) [inline CHANGES.md:13]: Can you please move this into the breaking changes section?

**Review by ankittrehan2000 — COMMENTED** [review]
@okrayum this looks good to merge, I just have the one comment. Thanks!

File: CHANGES.md (hunk 1 above)
Comment (okrayum) [reply CHANGES.md]: Hello @ankittrehan2000
I adjusted the CHANGES.md and also removed the following additional jitter code from VoxelRenderResources.js

if (primitive._jitter) {
    shaderBuilder.addDefine("JITTER", undefined, ShaderDestination.FRAGMENT);
  }
  
  Thank you for your feedback!

**Comment by ankittrehan2000** [comment]
Thanks @okrayum

**Comment by okrayum** [comment]
@ankittrehan2000 you're welcom

**Pull request merged**